.eslintrc.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
   settings: { react: { version: 'detect' } },
   plugins: ['react', , '@typescript-eslint'],
   rules: {
+    camelcase: ['error', { allow: ['_ONLY_FOR_TESTING'] }],
```

Comment:
🤷 is this a good idea? functions only to be used for testing having this suffix..?